OpenSim/Common/DebugUtilities.cpp

```diff
@@ -38,8 +38,7 @@ void Fatal_Error(const char* msg, const char* function, const char* file,
                                   "line = {})", msg, function, file, line);
     log_critical(str);
     throw std::runtime_error(str);
-    assert(false);
-    exit(1);
+    std::terminate();
```

Comment:
This is a drive-by edit, but is included here because:

- The original code makes no logical sense anyway (the thread will stop on the `throw`)
- The `assert(false)` is surplus, turns up in regex searches for bare `assert`s, and is compiler-toggleable
- The `exit` (and my `std::terminate`) are also surplus, but I wasn't sure whether something funny could be going on (e.g. disabling exceptions or something unusual), so I just replaced the `exit`, with the more C++stdlib-friendly `std::terminate()`